R/FilterStateChoices.R

```diff
@@ -494,7 +520,7 @@ ChoicesFilterState <- R6::R6Class( # nolint
             )
           } else {
             private$observers$selection <- observeEvent(
-              ignoreNULL = FALSE, # it's possible that nothing is selected
+              ignoreNULL = private$init_null, # init_null toggle for multiple var.
```

Comment:
I think we should avoid no-selection when `multiple = FALSE`. Default behaviour of the filter-panel is as follows:
If `selected` is not specified then `selected` is set to `choices`, With `multiple = FALSE` we need to initialize filter with `selected = choices[1]`. `selected = NULL` when `multiple = FALSE` should be forbidden.

Reply:
The variable private$init_null is set to TRUE when private$multiple is set to FALSE. This is because in the observeEvent for input$selection , the radio button, something has to be selected for input$selection, and it cannot be NULL. However, in the checkbox, it can be NULL. Hence, the variable private$init_null is used.

Reply:
I have modified the code to ensure that if "multiple = FALSE" and no selection is made (selected = NULL), the first available choice will be automatically selected in this scenario.